.clang-format

```diff
@@ -20,25 +20,10 @@ AlwaysBreakBeforeMultilineStrings: false
 AlwaysBreakTemplateDeclarations: true
 BinPackArguments: false
 BinPackParameters: false
-BraceWrapping:
-  # AfterCaseLabel was added in clang9
-  #AfterCaseLabel:        true
-  AfterClass:            true
-  AfterControlStatement: true
-  AfterEnum:             true
-  AfterFunction:         true
-  AfterNamespace:        false
-  AfterStruct:           true
-  AfterUnion:            true
-  AfterExternBlock:      true
-  BeforeCatch:           true
-  BeforeElse:            true
-  IndentBraces:          false
-  SplitEmptyFunction:    true
-  SplitEmptyRecord:      true
-  SplitEmptyNamespace:   true
 BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterFunction: true
```

Comment:
Nit: why moving this out of alphabetic order?

Reply:
Ah, didn't realize it was ordered. I wanted to move it closer to the `Custom` bit that enables `BraceWrapping`

Reply:
No problem with that, I was just wondering why the move. Feel free to leave it as it is.